pages/confirm-installation-popup.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class ConfirmInstallationPopup {
```

Comment:
I would recommend to not use Popup, recently we introduced actors (#15), and popups do not have their own folder so it is confusing, a popup is a modal page anyway, a page where you cannot interact with the background, so no sense IMO, we have this somewhere else, so it is a good time to start to correct it, ie: ConfirmationInstallationPage.